bentoml/yatai/yatai_service_impl.py

```diff
@@ -149,7 +149,7 @@ def ApplyDeployment(self, request, context=None):
         except BentoMLException as e:
             logger.error("RPC ERROR ApplyDeployment: %s", e)
             return ApplyDeploymentResponse(status=e.status_proto)
-        except Exception as e:
+        except Exception as e:  # pylint: disable=broad-except
```

Comment:
consider just add `disable=broad-except` to line 60 of this file, after `# pylint: disable=unused-argument`? That will disable for this entire code block